pyinfra/operations/brew.py

```diff
@@ -190,13 +198,19 @@ def tap(src, present=True):
     if present:
         if is_tapped:
             host.noop("tap {0} already exists".format(src))
-        else:
-            yield "brew tap {0}".format(src)
-            taps.append(src)
+            return
 
-    elif not present:
-        if is_tapped:
-            yield "brew untap {0}".format(src)
-            taps.remove(src)
-        else:
-            host.noop("tap {0} does not exist".format(src))
+        cmd = "brew tap {0}".format(src)
```

Comment:
let me know if this guard clause refactoring isn't appreciated -- no hard feelings about reverting to just the `url` change

Reply:
Much appreciated, good improvement 👍